crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -159,7 +159,7 @@ impl VM {
     }
 
     // SSTORE operation
-    // TODO: add gas consumption
+    // TODO: add gas REFUNDS
```

Comment:
If this is still pending, open an issue, and reference it here.

Reply:
🫡

Reply:
Done in [1ff2d77](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1029/commits/1ff2d77272f33be8879304c3e849b0b39c95df9d).